### PR TITLE
Feature/as 41 implement cookies page cm subtasks

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-41

## Description of change
Extract cookies implementation from FWA, move to Common package, and share with Questionnaire. 

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Comment by Russ-Clarke** [comment]
Hi Chris. Do you know why the builds are failing? Particularly the npm ci commands?

**Comment by a6software** [comment]
> Hi Chris. Do you know why the builds are failing? Particularly the npm ci commands?

Hi Russ, yes, sorry, it's a work in progress, not yet ready for review. I wanted to push to save / show my progress, though was aware it still needed work.

On with it currently, sorry for the confusion.

File: docker-compose.yml [hunk 1]
```diff
@@ -147,10 +149,11 @@ services:
     image: node:14-alpine
     environment:
       APPEALS_SERVICE_API_URL: http://appeals-service-api:3000
+      # one year
+      COOKIE_EXPIRES_TIMER_IN_MS: 31536000000
       DOCUMENTS_SERVICE_API_URL: http://document-service-api:3000
       PDF_SERVICE_API_URL: http://pdf-service-api:3000
       APPEAL_REPLY_SERVICE_API_URL: http://appeal-reply-service-api:3000
-      DOCUMENTS_SERVICE_API_URL: http://document-service-api:3000
```
Comment (a6software) [inline docker-compose.yml:153]: duplicate of line 150

File: packages/common/src/views/cookies.njk [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 {% block head %}
   {{ super() }}
-  <script src="/public/javascripts/cookiePreferences.bundle.js" defer></script>
+  <script src="/public/common/javascripts/cookiePreferences.bundle.js" defer></script>
```
Comment (a6software) [inline packages/common/src/views/cookies.njk:8]: to differentiate between local and `@pins/common` scripts

File: packages/common/tests/controllers/cookies.test.js [hunk 3]
```diff
@@ -40,7 +42,7 @@ describe('controllers/cookies', () => {
 
   describe('getCookies', () => {
     beforeEach(() => {
-      getPreviousPagePath.mockImplementation(() => fakePreviousPage);
+      // getPreviousPagePath.mockImplementation(() => fakePreviousPage);
```
Comment (a6software) [inline packages/common/tests/controllers/cookies.test.js:45]: remove

**Review by pbradley-f4 — APPROVED** [review]

**Pull request merged**